backend/routes/user.routes.ts

```diff
@@ -0,0 +1,60 @@
+import express = require('express');
+import { User } from '../models/user.models';
+import { auth } from '../middleware/auth';
+
+export const user3Routes = express.Router();
```

Comment:
I think this is Lucas' code, not yours, but I'm curious about the magic '3' here. Should we be embedding 'magic numbers' (a set number of possible routes, for example) into our code?

Reply:
That has been changed. I wrote 3 different variations of routes and functions, deleted 2 and forgot to change the final product variable name. Nothing more than that.